.github/workflows/deploy-website.yml

```diff
@@ -17,6 +17,7 @@ on:
   push:
     branches:
       - "master"
+      - "main"
```

Comment:
seems the ClickHouse org does not enable the GitHub Pages, you may need to find out another place to host the docs.

Reply:
We will probably move it to ClickHouse docs 
Just to verify you are referring to the website

Reply:
yes, the website, the previous link is https://housepower.github.io/spark-clickhouse-connector, is not available after the transfer.